openloops.spec

```diff
@@ -24,8 +24,8 @@ EOF
 export SCONSFLAGS="-j %{compiling_processes}"
 ./openloops update --processes generator=0
 ./openloops libinstall lhc.coll
-./openloops libinstall ppatt,ppatt_ew,ppattj,pplltt,pplltt_ew,ppllttj,ppllttj_ew,pplntt,pplnttj,ppwtt,ppwtt_ew,ppwttj,ppwttj_ew,ppztt,ppztt_ew,ppzttj,pphtt,pphtt_ew,pphttj
```

Comment:
@SiewYan is the line above still needed?

Reply:
Not really, the user defined collection should consist ALL lhc.coll. But in case of overlapping lib, the installation will update/skip the lib.